content/docs/user-guide/components/reference/physx/shape-collider.md

```diff
@@ -1,92 +1,78 @@
 
-****Trigger****
-When enabled, this shape collider functions as a trigger. A trigger performs a quick overlap test and does not apply forces or return contact point information. Use this to speed up PhysX computations where a simple overlap between colliders is sufficient.
-Trigger Area components are legacy components and cannot be used with PhysX Shape Collider.
+* The shape information defined by the Shape component is used elsewhere in code or script. For example, the shape defines another volume, such as an audio volume or fog volume, and you want to keep the collider geometry and volume synchronized.
+* You want to use a Shape component such as [Polygon Prism Shape](/docs/user-guide/components/reference/shape/polygon-prism-shape/) that is not provided by PhysX Collider.
+* You have existing Shape components and don't want to migrate them to use PhysX Collider components.
 
-****Simulated****
-When enabled, this shape collider will be part of the physics simulation.
+## Limitations
 
-****In Scene Queries****
-When enabled, this shape collider can be queried for raycasts, shapecasts and overlap.
-
-****Library (Physics Materials)****
-The physics material library of the project.
-
-****Slot (Physics Materials)****
-Choose a physics material for this shape collider.
+The PhysX Shape Collider component has some limitations compared to the PhysX Collider component:
 
-****Tag****
-Set a tag for this shape collider. Tags can be used to quickly identify components in script or code.
+* Only one Shape component can be used per entity, and so only one PhysX Shape Collider component is supported per entity. Any number of PhysX Collider components can also be used on the same entity, however.
+* The position and rotation of the PhysX Shape Collider component can't be offset relative to the entity position.
 
-****Rest offset****
-PhysX bodies come to rest separated by the sum of their rest offset values. The **Rest offset** value must be less than the **Contact offset** value. Valid values rage from **-Infinity** to **50**.
+## Properties 
 
-****Contact offset****
-PhysX bodies generate contacts when they are within the sum of their contact offset values. The **Contact offset** value must be greater than the **Rest offset** value. Valid values rage from **0** to **50**.
+![PhysX Shape Collider component interface](/images/user-guide/components/reference/physx/ui-shape-collider-a.png)
 
-****Draw collider****
-Render this shape collider in the viewport. Enabled by default.
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Collision Layer** | The collision layer that's assigned to the collider. | Any collision layer | `Default` |
+| **Collides With** | The collision group containing the layers that this collider collides with. | Any collision group | `All` |
```

Comment:
Link "collision layer" and "collision group" in the 3rd column to their respective pages in the User Guide, as suggested for the PhysX Collider.